subworkflows/local/bin_qc.nf

```diff
@@ -81,57 +79,28 @@ workflow BIN_QC {
         /*
          * BUSCO
          */
+        busco_lineage = params.busco_auto_lineage_prok ? 'auto_prok' : 'auto'
+
         if (!ch_busco_db.isEmpty()) {
             if (ch_busco_db.extension in ['gz', 'tgz']) {
-                // Expects to be tar.gz!
-                BUSCO_DB_PREPARATION(ch_busco_db)
-                ch_db_for_busco = BUSCO_DB_PREPARATION.out.db.map { meta, db ->
-                    [[id: meta, lineage: 'Y'], db]
+                BUSCO_UNTAR([[id: ch_busco_db.getSimpleName()], ch_busco_db])
```

Comment:
No changes needed for this PR: Ok this tripped me up at first as I was like, you shouldn't do a `getSimpleName` on a channel but then I realised it isn't a channel just a file...

I think we should come up with new conventions how to prefix things like this...

Maybe

ch_: for channels
fl_: for files
val_: for variables (Strings, integers etc)

What do you guys think? Can bring it up in the rest of the community too if you think it's important

Reply:
I like it also - but might be worth waiting to see how DSL3 is going to look as AFAIK it's getting types